examples/jsm/misc/Volume.js

```diff
@@ -427,7 +427,7 @@ class Volume {
 	/**
 	 * Call repaint on all the slices extracted from this volume
 	 *
-	 * @see VolumeSlice.repaint
+	 * @see {@link VolumeSlice.repaint}
```

Comment:
Can we fix this to `VolumeSlice#repaint`? I'm not sure the dot syntax is valid...